src/Generator/Generators/CLI/CLIMarshal.cs

```diff
@@ -40,13 +40,15 @@ public override bool VisitTagType(TagType tag, TypeQualifiers quals)
             return decl.Visit(this);
         }
 
+        private static int _arrayNum = 0;
```

Comment:
Do not prefix fields, static or instance, with an underscore. We don't do that in this project.